src/app/frame/page.tsx

```diff
@@ -1,10 +1,130 @@
-import MyButton from '@/components/MyButton';
+'use client';
+
+import Image from 'next/image';
+import Link from 'next/link';
+import { useState } from 'react';
+import PreviousPage from '@/components/PreviousPage';
+import { BASIC_FRAME_DATA, PREMIUM_FRAME_DATA } from '@/constants';
+import { FrameProps } from '@/interfaces';
+import SVGDownload from '@/styles/icons/download.svg';
+import SVGGoToList from '@/styles/icons/gotolist.svg';
+import DownloadImage from './_components/DownloadImage';
+
+function SelectFrame({ circle, description, onClick }: FrameProps) {
+  return (
+    <button type="button" onClick={onClick} className="cursor-pointer">
+      <Image
+        src={circle}
+        alt={description}
+        width={40}
+        height={40}
+        priority
+        className="relative"
+      />
+    </button>
+  );
+}
+
+function FrameSelectView() {
+  const [colorOfCircle, setColorOfCircle] = useState<string>('');
+  const [isPremiumSelected, setIsPremiumSelected] = useState<boolean>(false);
+
+  const frametype = isPremiumSelected ? PREMIUM_FRAME_DATA : BASIC_FRAME_DATA;
+  const frameWidth = colorOfCircle === '/premiumframe2.png' ? 283 : 239;
+  const frameHeight = colorOfCircle === '/premiumframe2.png' ? 332 : 290;
+  const imageSrc = '/resultsample.png'; // 생성된 이미지(임시)
+
+  const onCaptureClick = () => {
+    DownloadImage({ colorOfCircle, imageSrc, frameWidth, frameHeight });
+  };
 
-export default function FrameSelectView() {
   return (
-    <div>
-      <h1>This is 프레임 선택 Page</h1>
-      <MyButton name="다운로드" target="/" />
+    <div className="flex w-full flex-col items-center justify-center">
+      <div className="flex w-full flex-row justify-between px-4">
+        <PreviousPage target="/" />
+        <Link href="/list" className="pr-4">
+          <SVGGoToList />
+        </Link>
+      </div>
+      <h1 className="text-xl">AI 프로필이 완성!</h1>
+      <h2 className="py-1 font-sfpro text-2xs font-bold text-white">
+        사진과 어울리는 프레임을 선택해주세요 😀
+      </h2>
+      <div className="text-2sx mb-6 mt-4 flex h-[35px] w-[183px] flex-row items-center justify-evenly rounded-full bg-white font-sfpro font-bold">
+        {isPremiumSelected ? (
```

Comment:
프리미엄 여부에 따라서 달라지는 부분이 className이랑 내부 택스트 뿐이라면 심항연산자를 사용하는것은 어떻게 생각하나요?

코드가 짧아질수 있을거 같아서 제안드립니다.
```suggestion
<div className=`flex h-[27px] w-[86px] items-center justify-center rounded-full ${isPremiumSelected ? "bg-white text-primary-middlegray" ? "bg-primary-darkblue text-white" }`>
</div>
```
__참고 URL__
https://stackoverflow.com/questions/74092110/how-do-i-use-ternary-operator-within-classname

Reply:
좋은 제안 감사합니다! 수정했습니다